.prettierignore

```diff
@@ -16,3 +16,6 @@ common/lib/**/lib
 **/_api-extractor-temp/*
 **/api-report/*
 **/*.api.md
+
+# End-to-End Test
```

Comment:
I would update these comments to say something like 
```suggestion
# Generated type-tests
```

Reply:
And I see a couple of entries where the comment is missing - I would advocate for adding it (even if there aren't other comments in those files)

Reply:
Looks like you missed this one. But the others look good :)

Reply:
Thanks for the catch! 😄